examples/spring-boot-minimal-kotlin/src/main/kotlin/example/springframework/boot/minimal/kotlin/Main.kt

```diff
@@ -0,0 +1,11 @@
+package example.springframework.boot.minimal.kotlin
+
+import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.boot.runApplication
+
+@SpringBootApplication
+class Main
+
+fun main(args: Array<String>) {
+    runApplication<Main>(*args)
+}
```

Comment:
Question: Is this conventional not to indent `fun main()` in Kotlin?

Reply:
Yes, Kotlin calls it 'top-level functions'. 😮

Reply:
`main()` is NOT part of the `Main` class with this syntax. `main()` is part of the "generated" `<filename>Kt` class i.e. in this case `MainKt.class`.

Reply:
Yes. So I wrote [this](https://github.com/line/armeria/blob/master/examples/grpc-kotlin/build.gradle.kts#L22) when creating `grpc-kotlin`. 😄 

I just written it as guided by [spring-boot](https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-kotlin-api-runapplication).